drivers/display/display_st7789v_tl019fqv01.c

```diff
@@ -17,7 +17,18 @@ void st7789v_lcd_init(struct st7789v_data *p_st7789v)
 	 * display is 320x170, and so the effective LCD is offset
 	 * by 35 "rows".
 	 */
-	st7789v_set_lcd_margins(p_st7789v, 0, 35);
```

Comment:
It does not make sense, this file is for TL019FQV01 display, everything here is hardcoded, it is not generic. Working on support for [Waveshare display](https://github.com/zephyrproject-rtos/zephyr/pull/20570#issuecomment-553656955), I will remove it and replace with byte arrays in DT.

Reply:
Agree that it's corner implementation. Are you willing to remove whole file (display_st7789v_tl019fqv01.c)?

Reply:
It will be moved to display_st7789v.c and should obtain the values from DT. I think we should not have a display_st7789v_xVARIANTx.c per display.

Reply:
Great. Sincerely, i do not like these display_name_variant.c implementations. Thank you for taking care of this. Closing PR